Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -69,15 +69,21 @@ public class JavaScriptLifter: ComponentBase, Lifter {
         }
     }
 
-    private func lift(_ program: Program, withOptions options: LiftingOptions, withPolicy policy: InliningPolicy) -> String {
+    //Support function allowing compiler development to lift arbitrary protobufs
+    public func ilSupportLift(_ program: Program) -> String {
```

Comment:
Is the issue here that the JavaScriptLifter normally requires a `Fuzzer` instance to operate? And in this case the `fuzzer` and `logger` members will be `nil` now? That's probably not great since it could lead to crashes when one of those is used in other places in the future...
I think we should fix that properly. For now it should be enough to pass in the Environment to the constructor and turn the `logger.fatal` into a `fatalError`. WDYT?

Reply:
Yes, that's the issue I ran into. My goal is minimizing the amount of setup and options that FuzzIlTool has to set up, which don't affect how it operates. I think the change you suggested made sense, and have pushed that up!

Reply:
Cool! With that, this function is no longer needed, is it?